legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabReviewApi.java

```diff
@@ -122,8 +157,25 @@ else if (revisionIds instanceof Set)
             {
                 // if no revision ID is specified we will use the default merge request API from Gitlab to take advantage of the filter
                 MergeRequestFilter mergeRequestFilter = new MergeRequestFilter()
-                        .withProjectId(gitLabProjectId.getGitLabId())
                         .withState(mergeRequestState);
+
+                if (projectId != null)
+                {
+                    mergeRequestFilter.setProjectId(projectId);
+                }
+
+                if (assignedToMe != null && assignedToMe)
+                {
+                   try
+                   {
+                       mergeRequestFilter.setAuthorId(Integer.parseInt(getCurrentUser()));
+                   }
+                   catch (NumberFormatException e)
+                   {
+                       LOGGER.debug("Cant format User to string " + getCurrentUser());
+                   }
+                }
```

Comment:
getCurrentUser() will not return an appropriate value here. You will need to query GitLab to get the right kind of author id.